### ISSUE TITLE
Feature Request - Sync

### ISSUE THREAD

**Issue by clefus** [RasPlex/RasPlex#521]
Rasplex should add sync to their list of features.  This would set it apart from many of the existing options out there....it could be highly effective and useful in a variety of situations.

Thanks for reading/considering.

[Tell us about your issue, steps to reproduce, expected and actual behavior]
[Always attempt to reproduce your issue in the default skin]
[Enable debug logging in preferences and insert a link to your log-file]

**RPi model**: [raspberry pi model]
**RasPlex**: [version and skin]
**PMS**: [version and operating system]
**Network**: [ethernet/wifi]

**Comment by Ruvetuve** [comment]
If it can sync to an attached Hard Disk, this feature would be great for viewing Plex content on the road. Thinking about plugging it in on hotel TV's etc.

**Comment by clefus** [comment]
I agree Ruvetuve.  While copying them to the SD card is possible....I think that sync should still be implemented.

It could also benefit those with network problems or slow wifi in their homes.  But, at the end of the day - the designers need to know that not all users of rasplex are savy enough to copy to SD etc.  (IE my wife lol).  I like the KISS principle.....keep it simple....

I truly believe this one feature would set RASPLEX out above some of the other players that are out there....and it would have many practical uses for people.  On the road for kids, hotels, portable kiosks, and so much more.

**Comment by leprasmurf** [comment]
:+1: this feature would be immensely useful.  Especially for clients that rely on wifi.  The server can transcode for simplified playback before sync'ing, that removes the need for transcoding by the Pi.

**Comment by ovizii** [comment]
I agree but unfortunately RasPlex is built upon Plex Home Theater (PHT) and PHT had no sync option so it looks like its not going to happen too soon.

**Comment by vbtrek** [comment]
I'd like sync too, because I have an RPI stuck to the back of the TV with a USB HDD also stuck to the back of the TV which we use at home and whilst travelling in the caravan. Any other solutions available for RPI other than RasPlex?

**Comment by WebSpider** [comment]
OpenPHT on raspbian with Plex Media Server on the same box should work

On 8 April 2018 10:47:40 CEST, Derek Smith <notifications@github.com> wrote:
>I'd like sync too, because I have an RPI stuck to the back of the TV
>with a USB HDD also stuck to the back of the TV which we use at home
>and whilst travelling in the caravan. Any other solutions available for
>RPI other than RasPlex?
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/RasPlex/RasPlex/issues/521#issuecomment-379532526

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by justinglow** [comment]
Would also love syncing locally for playing offline.
